PyBaseConverter.py

```diff
@@ -15,6 +15,11 @@
 ## You should have received a copy of the GNU General Public License
 ## along with PyBaseConverter.  If not, see <http://www.gnu.org/licenses/>.
 
+"""A base converter written in Python.
+
+The main purpose of this code was to test PyQt5.
```

Comment:
This is of no concern to whoever downloads this project. Please use the extra lines in the docstring to say something more about the module.